scripts/addr2line.py

```diff
@@ -60,48 +69,57 @@ def __init__(self, binary, concise=False, cmd_path="addr2line"):
             print('{}'.format(s))
 
         options = f"-{'C' if not concise else ''}fpia"
-        self._input = subprocess.Popen(
+        self._input_proc = subprocess.Popen(
             [cmd_path, options, "-e", self._binary],
             stdin=subprocess.PIPE,
             stdout=subprocess.PIPE,
             universal_newlines=True,
         )
         if concise:
-            self._output = subprocess.Popen(
+            self._output_proc = subprocess.Popen(
                 ["c++filt", "-p"],
-                stdin=self._input.stdout,
+                stdin=self._input_proc.stdout,
                 stdout=subprocess.PIPE,
                 universal_newlines=True,
             )
         else:
-            self._output = self._input
+            self._output_proc = self._input_proc
 
         # If a library doesn't exist in a particular path, addr2line
         # will just exit.  We need to be robust against that.  We
         # can't just wait on self._addr2line since there is no
         # guarantee on what timeout is sufficient.
-        self._input.stdin.write('\n')
-        self._input.stdin.flush()
-        res = self._output.stdout.readline()
+        self._input.write('\n')
+        self._input.flush()
+        res = self._output().readline()
         self._missing = res == ''
 
+    @property
```

Comment:
the last nit: can we make similar changes to `self._output`, so they are symmetric ?

Reply:
Thanks for the suggestion, done in most recent push.